console/console-init/ui/src/modules/device/components/DeviceFilter/AddedDateFilterSection.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import {
+  TextInput,
+  InputGroup,
+  TextContent,
+  DropdownPosition,
+  InputGroupText
+} from "@patternfly/react-core";
+import { css, StyleSheet } from "@patternfly/react-styles";
+import { IDeviceFilter } from "./DeviceFilter";
+import { DropdownWithToggle } from "components";
+import { ISelectOption } from "utils";
+import { CalendarAltIcon } from "@patternfly/react-icons";
+
+const styles = StyleSheet.create({
+  time_input_box: {
+    padding: 20,
+    marginRight: 10
+  },
+  dropdown_align: { display: "flex", marginRight: 10 },
+  dropdown_toggle_align: { flex: "1" }
+});
+
+interface IAdddedDateFilterSectionProps {
+  filter: IDeviceFilter;
+  setFilter: (filter: IDeviceFilter) => void;
+}
+const AdddedDateFilterSection: React.FunctionComponent<IAdddedDateFilterSectionProps> = ({
```

Comment:
There is a typo here in the component name. Also for the naming, to me, just 'DateFilterSection' makes more sense, what do you think?

Reply:
done